test/tests/utils/libtorch_utils/vector-conversion.i

```diff
@@ -0,0 +1,24 @@
+[Mesh]
+  [gmg]
+    type = GeneratedMeshGenerator
+    dim = 1
+  []
+[]
+
+[VectorPostprocessors]
+  [test]
+    type = LibtorchVectorConversionTest
```

Comment:
I'd like to keep this as is because GTEST cannot compare `torch::Tensors` and `std::vectors`. If I coded the comparison I would just use the same lines as in the conversion functions so I would practically test the functions with themselves.

Reply:
Now that I get to think about it, I think I'll convert this to a unit test.